src/modules/mhct-lookup.js

```diff
@@ -214,6 +217,59 @@ async function formatMice(isDM, mouse, opts) {
     reply += '```\n' + `HTML version at: ${target_url}`;
     return reply;
 }
+/**
```

Comment:
Generally it's helpful (for us humans) to separate methods with a newline
```suggestion

/**
```